src/controllers/auth.controller.js

```diff
@@ -84,6 +85,36 @@ class AuthController {
     ResponseService.setSuccess(200, 'Successful logout');
     return ResponseService.send(res);
   }
+
+  /**
+    * @param {Request} req
+    * @param {Response} res
+    * @returns {FindUser} this function finds if user exist in database
+    */
+  static async findUser(req, res) {
```

Comment:
I think that this function name is misleading. It finds a user but also send an email to him. so find user might relate to just fetching from DB. Why not call it another name like `sendEmail`